dataservice/api/common/pagination.py

```diff
@@ -0,0 +1,15 @@
+from flask import request, current_app
+from functools import wraps
+
+
+def paginated(f):
+
+    @wraps(f)
+    def paginated_wrapper(*args, **kwargs):
+        def_size = current_app.config['DEFAULT_PAGE_SIZE']
+        max_size = current_app.config['MAX_PAGE_SIZE']
+        limit = min(request.args.get('size', def_size, type=int), max_size)
```

Comment:
I think we should name the results per page parameter consistently. In the query params its named `size` but in the responses that come back its called `limit`. Lets choose one or the other....but personally I like `limit_per_page` or `results_limit`, even though those are not conventional. I like them bc their meanings are clearer to me than `limit` or `size`